src/js/references/components/__tests__/ReferenceList.test.tsx

```diff
@@ -109,22 +115,25 @@ describe("<ReferenceList />", () => {
             renderWithRouter(<ReferenceList />, state, history);
 
             expect(await screen.findByText("References")).toBeInTheDocument();
+            expect(await screen.findByRole("link", { name: "clone" })).toBeInTheDocument();
             await userEvent.click(screen.getByRole("link", { name: "clone" }));
-
             await userEvent.click(screen.getByRole("button", { name: "Clone" }));
 
             getReferencesScope.done();
             cloneReferenceScope.done();
         });
 
         it("handleSubmit() should mutate with changed input", async () => {
+            const permissions = createFakePermissions({ create_ref: true });
+            const account = createFakeAccount({ permissions: permissions });
+            mockAPIGetAccount(account);
             const getReferencesScope = mockApiGetReferences([references]);
             const cloneReferenceScope = mockApiCloneReference("newName", `Cloned from ${references.name}`, references);
             renderWithRouter(<ReferenceList />, state, history);
 
             expect(await screen.findByText("References")).toBeInTheDocument();
+            expect(await screen.findByRole("link", { name: "clone" })).toBeInTheDocument();
             await userEvent.click(screen.getByRole("link", { name: "clone" }));
```

Comment:
```suggestion
            await userEvent.click(await screen.findByRole("link", { name: "clone" }));
```
afaik you can merge these two. If the component does not exists then this should throw an error

Reply:
Yes, good idea